partner_communication_switzerland/models/partner_communication.py

```diff
@@ -342,8 +344,18 @@ def _get_new_dossier_attachments(self, correspondence=True, payment=True):
                 'group_id.contract_ids').filtered(
                 lambda s: s.state == 'active')
 
+        make_payment_pdf = True
+
+        groupe_ids = sponsorships.mapped('group_id')
+
+        account_payment_mode_list = account_payment_mode_obj.search(['|', ('name', 'like', 'Direct'),('name', 'like', 'LSV')])
```

Comment:
Line is too long, and you should add a space between the conditions. Also change "Direct" by "Direct Debit"

Reply:
You could also rename variable to `lsv_dd_modes`